plutus-scb/src/Plutus/SCB/Effects/ContractTest.hs

```diff
@@ -126,23 +126,22 @@ doContractUpdate contract payload = do
         $ fromString
         $ traverse (JSON.parseEither JSON.parseJSON) payload
     logDebug $ Request $ pretty request
-    response <-
-        either (throwError . OtherError) (pure . mkResponse)
-        $ ContractState.insertAndUpdateContract contract request
+    let response = mkResponse $ ContractState.insertAndUpdateContract contract request
```

Comment:
Again, missing rethrowing?

Reply:
Here it is also better not to throw the error, because then we get more information with the result.